modules/serverless-functions-setting-custom-volume-size.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-on-cluster-builds.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-functions-setting-custom-volume-size_{context}"]
+= Setting custom volume size
+
+For projects that require a volume with a larger size to build, you might need to customize the persistent volume claim (PVC) when building on the cluster. The default PVC size is 256 mebibytes.
+
+.Prerequisites
+
+* {pipelines-title} must be installed on your cluster.
+
+* You have installed the OpenShift (`oc`) CLI.
```

Comment:
There's an attribute for this now (assuming it works in serverless docs :thinking:)
```suggestion
* You have installed the {oc-first}.
```

Reply:
No, we don't have this attribute.